cmd/prysmctl/validator/withdraw.go

```diff
@@ -0,0 +1,150 @@
+	}
+	return callWithdrawalEndpoints(ctx, beaconNodeHost, setWithdrawalAddressJsons)
+}
+
+func getWithdrawalMessagesFromPathFlag(c *cli.Context) ([]*apimiddleware.SignedBLSToExecutionChangeJson, error) {
+	setWithdrawalAddressJsons := make([]*apimiddleware.SignedBLSToExecutionChangeJson, 0)
+	foundFilePaths, err := findWithdrawalFiles(c.String(PathFlag.Name))
+	if err != nil {
+		return setWithdrawalAddressJsons, errors.Wrap(err, "failed to find withdrawal files")
+	}
+	for _, foundFilePath := range foundFilePaths {
+		b, err := os.ReadFile(filepath.Clean(foundFilePath))
+		if err != nil {
+			return setWithdrawalAddressJsons, errors.Wrap(err, "failed to open file")
+		}
+		var to []*apimiddleware.SignedBLSToExecutionChangeJson
+		if err := json.Unmarshal(b, &to); err != nil {
+			log.Warnf("provided file: %s, is not a list of signed withdrawal messages", foundFilePath)
+			continue
+		}
+		setWithdrawalAddressJsons = append(setWithdrawalAddressJsons, to...)
+	}
+	if len(setWithdrawalAddressJsons) == 0 {
+		return setWithdrawalAddressJsons, errors.New("the list of signed requests is empty")
+	}
+	return setWithdrawalAddressJsons, nil
+}
+
+func callWithdrawalEndpoints(ctx context.Context, host string, request []*apimiddleware.SignedBLSToExecutionChangeJson) error {
+	client, err := beacon.NewClient(host)
+	if err != nil {
+		return err
+	}
+	if err := client.SubmitChangeBLStoExecution(ctx, request); err != nil {
+		return err
+	}
+	log.Infof("Successfully published messages to update %d withdrawal addresses.", len(request))
+	return checkIfWithdrawsAreInPool(ctx, client, request)
+}
+
+func checkIfWithdrawsAreInPool(ctx context.Context, client *beacon.Client, request []*apimiddleware.SignedBLSToExecutionChangeJson) error {
+	log.Info("verifying requested withdrawal messages known to node...")
+	poolResponse, err := client.GetBLStoExecutionChanges(ctx)
+	if err != nil {
+		return err
+	}
+	requestMap := make(map[string]string)
+	for _, w := range request {
+		requestMap[w.Message.ValidatorIndex] = w.Message.ToExecutionAddress
+	}
+	for _, resp := range poolResponse.Data {
+		value, found := requestMap[resp.Message.ValidatorIndex]
+		if found && value == resp.Message.ToExecutionAddress {
+			log.WithFields(log.Fields{
+				"validator_index":    resp.Message.ValidatorIndex,
+				"execution_address:": resp.Message.ToExecutionAddress,
+			}).Info("set withdrawal address message was found in the node's operations pool.")
+			delete(requestMap, resp.Message.ValidatorIndex)
+		}
+	}
+	if len(requestMap) != 0 {
+		for key, address := range requestMap {
+			log.WithFields(log.Fields{
+				"validator_index":    key,
+				"execution_address:": address,
+			}).Warn("set withdrawal address message not found in the node's operations pool.")
+		}
+		log.Warn("please check before resubmitting. set withdrawal address messages that were not found in the pool may have been included into a block.")
```

Comment:
```suggestion
		log.Warn("Please check before resubmitting. Set withdrawal address messages that were not found in the pool may have been already included into a block.")
```
Also it's not clear to me what `Please check` means. Check what?